File: intelligencecity_manager_interface/src/main/java/com/shanghaiuniversity/manager/service/MinitorerService.java
package com.shanghaiuniversity.manager.service;

import com.ShanghaiUniversity.entity.PageResult;
import com.ShanghaiUniversity.pojo.TbMinitorer;

import java.util.List;

/**
 * 服务层接口
 *
 * @author Administrator
 */
public interface MinitorerService {

    /**
     * 返回全部列表
     *
     * @return
     */
    public List<TbMinitorer> findAll();

    /**
     * 返回分页列表
     *
     * @return
     */
    public PageResult findPage(int pageNum, int pageSize);

    /**
     * 增加
     */
    public void add(TbMinitorer minitorer);

    /**
     * 修改
     */
    public void update(TbMinitorer minitorer);

    /**
     * 根据ID获取实体
     *
     * @param id
     * @return
     */
    public TbMinitorer findOne(Long id);

    /**
     * 批量删除
     *
     * @param ids
     */
    public void delete(Long[] ids);

    /**
     * 分页
     *
     * @param pageNum  当前页 码
     * @param pageSize 每页记录数
     * @return
     */
    public PageResult findPage(TbMinitorer minitorer, int pageNum, int pageSize);

}

File: intelligencecity_manager_web/src/main/webapp/assets/js/service/minitorerService.js
//服务层
app.service('minitorerService',function($http){
	    	
	//读取列表数据绑定到表单中
	this.findAll=function(){
		return $http.get('../minitorer/findAll.do');		
	}
	//分页 
	this.findPage=function(page,rows){
		return $http.get('../minitorer/findPage.do?page='+page+'&rows='+rows);
	}
	//查询实体
	this.findOne=function(id){
		return $http.get('../minitorer/findOne.do?id='+id);
	}
	//增加 
	this.add=function(entity){
		return  $http.post('../minitorer/add.do',entity );
	}
	//修改 
	this.update=function(entity){
		return  $http.post('../minitorer/update.do',entity );
	}
	//删除
	this.dele=function(ids){
		return $http.get('../minitorer/delete.do?ids='+ids);
	}
	//搜索
	this.search=function(page,rows,searchEntity){
		return $http.post('../minitorer/search.do?page='+page+"&rows="+rows, searchEntity);
	}    	
});
